### PR TITLE
PS1 connected to database

### PR DESCRIPTION
**Pull request by pjchong91**
Connects to database

### CODE REVIEW

File: js/lib/formatUpdatesData.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
```
Comment (siddharthparmar7) [inline js/lib/formatUpdatesData.js]: Delete this file and put this function in the helpers.js file

File: js/screens/Profile/ProfileContainer.js [hunk 2]
```diff
@@ -1,37 +1,45 @@
 import React, { Component } from 'react';
 import Profile from './Profile';
 import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import { Query } from 'react-apollo';
+import { Text } from 'react-native';
 
+const CHILD_DATA = gql`
```
Comment (siddharthparmar7) [inline js/screens/Profile/ProfileContainer.js:8]: Stick to the pattern you chose as a group. Either write your query in the `<Query />` component or create a const or just have a file for all of your queries.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes. Thank you.

File: js/screens/Profile/ProfileContainer.js (hunk 2 above)
Comment (pjchong91) [reply js/screens/Profile/ProfileContainer.js:8]: We are going to create a const for the queries in their respective containers.  I will change the other files to match this one.

Comment (siddharthparmar7) [reply js/screens/Profile/ProfileContainer.js:8]: Thank you!

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
I can still see this `FormatLearningCurriculum.js` file on your github branch. Please delete it.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**